app/controllers/goldencobra/sessions_controller.rb

```diff
@@ -54,5 +50,14 @@ def logout
 
     def register
     end
+
+    private
+
+    def usermodel
+      return false if params[:usermodel].blank?
+      return false unless %w(user visitor).include?(params[:usermodel].to_s.downcase)
```

Comment:
Wird denn der session_controller anderweitig verwendet als explizit für Golden Cobra Logiken? Können dann überhaupt andere Models als User und Visitor in Betracht kommen?